share/admin.py

```diff
@@ -213,6 +214,19 @@ class HarvestLogAdmin(admin.ModelAdmin):
     list_filter = ('status', SourceConfigFilter, )
     list_select_related = ('source_config__source', )
     readonly_fields = ('harvest_log_actions',)
+    actions = ('restart_tasks', )
+
+    STATUS_COLORS = {
+        'created': 'blue',
+        'started': 'cyan',
+        'failed': 'red',
+        'succeeded': 'green',
+        'rescheduled': 'goldenrod',
+        'defunct': 'purple',
```

Comment:
Ah, I'll add some comments next to the status definitions.

defunct would be if a task is started with an older version of a harvester/transformer. And I also just realized that we default to the latest version of everything so this case will never be hit... Whoops. Will remove.